routes/user.js

```diff
@@ -93,13 +93,20 @@ router.post(
   }),
 );
 
-router.get(
-  '/get_by_id/:id',
-  runAsyncWrapper(async function (req, res) {
+router.get('/get_by_id/:id', async (req, res) => {
```

Comment:
so best convention for naming routes would just be "/user/:id" - because we have /:id, we don't need to specify get_by_id